src/main/java/zd/zero/waifu/motivator/plugin/settings/WaifuMotivatorState.kt

```diff
@@ -0,0 +1,24 @@
+package zd.zero.waifu.motivator.plugin.settings
+
+class WaifuMotivatorState {
+
+  var isWaifuOfTheDayEnabled = true
+
+  var isStartupMotivationEnabled = true
+
+  var isStartupMotivationSoundEnabled = true
+
+  var isUnitTesterMotivationEnabled = true
+
+  var isUnitTesterMotivationSoundEnabled = true
+
+  var isMotivateMeEnabled = true
+
+  var isMotivateMeSoundEnabled = true
+
+  var isSayonaraEnabled = true
+
+  var disableInDistractionFreeMode = true
+
+  var version = "v0.0.0"
```

Comment:
Does it seem that we could move the `version` to `WaifuMotivator` instead? `version` shouldn't be mutable and move it to a constant class, similar to what we do in `WaifuMotivator.PLUGIN_ID`, or is there a reason behind this intent?

Reply:
That is just a default value new users will get. In order to check if the version has changed we need to know the previous version. That way when the plugin starts up, we have something to compare.

The actual version of the plugin comes from here
https://github.com/Unthrottled/waifu-motivator-plugin/blob/57266f20172eba366d7171dea5b48bb4ff034913/src/main/java/zd/zero/waifu/motivator/plugin/onboarding/UserOnboarding.kt#L23